.yo-rc.json

```diff
@@ -1,10 +1,34 @@
 {
-  "generator-jhipster-module": {
-    "promptValues": {
-      "githubName": "oktadeveloper",
-      "authorName": "Matt Raible",
-      "authorEmail": "matt.raible@okta.com",
-      "authorUrl": "https://developer.okta.com"
-    }
+  "generator-jhipster": {
+    "baseName": "ionic",
+    "cli": true,
+    "generators": {
+      "app": {
+        "command": false,
+        "priorities": ["prompting", "composing"],
+        "sbs": true,
+        "written": true
+      },
+      "ionic": {
+        "command": true,
+        "priorities": [
+          "initializing",
+          "configuring",
+          "loading",
+          "preparing",
+          "writing",
+          "writingEntities",
+          "postWriting",
+          "install",
+          "end"
+        ],
+        "sbs": true,
+        "written": true
+      }
+    },
+    "jhipsterVersion": "7.5.0",
+    "prettierDefaultIndent": 2,
+    "projectName": "JHipster Ionic Blueprint v2",
```

Comment:
Do we need to include the version number? Technically, it's 1.0 since the previous version was a module, not a blueprint. I like calling it 2.0 though!